### PR TITLE
Wrote the Overview, History, Analysis and Value Proposition paragraphs

### PR DESCRIPTION
**Pull request by ecornamu**
Note that I already wrote them on the main before learning that we needed to do PRs so I reverted them on the main and put them on this branch to PR, hence the high number of commits

### CODE REVIEW

File: template/2-history.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
+
+Social features were also a significant part of the PoC. Users could add friends within the app and send messages, fostering a sense of community and making it easier to plan group activities.
+
+From the V1 PoC, we learned that the core functionalities were well designed: it it easy to find and save activities. The multilingual support is crucial, as it broadens the app's appeal. However, we also identified areas that are needing improvement: more personalized recommendations and advanced filtering options for example.
+
+To transition from the PoC to a Minimum Viable Product, several key enhancements are necessary. First, we need to refine the recommendation engine to provide more personalized and accurate suggestions based on user preferences and activity history. Improving the filtering capabilities will also be crucial, allowing users to narrow down activities more effectively.
```
Comment (JeremyHugentobler) [inline template/2-history.md:13]: For this part, I would suggest you to add the fact that the app still lacks of activity. We might need to find an other source of data and combine it with OSM to have enough activities in all categories.

File: template/2-history.md [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
+
+The social features need to be expanded to include more interactive elements, such as activity sharing, group event planning, and integration with broader social media platforms. By addressing these areas, we can bridge the gap from a promising PoC to a robust and widely-adopted MVP, ultimately fulfilling Lasta’s mission to revolutionize the outdoor sports community in Switzerland.
```
Comment (JeremyHugentobler) [inline template/2-history.md]: Activity sharing is already present in and out the app.

File: template/3-analysis.md
Comment (JeremyHugentobler) [inline template/3-analysis.md:1]: I think that you forgot to address the

*What are the complementary products in the market today?*

File: template/2-history.md (hunk 1 above)
Comment (JeremyHugentobler) [reply template/2-history.md:13]: Maybe also adding some new sports

**Review by JeremyHugentobler — CHANGES_REQUESTED** [review]
Hi,

Thanks you for your work ! It is really well written and the content is intersting. I added some comments where I think changes are needed and would be happy to discuss about it if you do not agree on certain points.

++

File: template/3-analysis.md
Comment (ecornamu) [reply template/3-analysis.md:1]: Yeah I thought about it but I had nothing interesting to say besides listing them so I decided not to

**Review by JeremyHugentobler — APPROVED** [review]
Thank you for the changes

**Pull request merged**